downloads/github-actions-cheat-sheet.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: cheat-sheet
+redirect_to: false
+title: GitHub Actions Cheat Sheet
+byline: GitHub Actions enables you to create custom software development lifecycle workflows directly in your GitHub repository.
+leadingpath: ../../
+---
+
+{% capture colOne %}
+## About GitHub Actions
+GitHub Actions gives you the flexibility to build an automated software development lifecycle workflow. You can write individual tasks, called actions, and combine them to create a custom workflow which are custom automated processes that you can set up in your repository to build, test, package, release, or deploy any code project on GitHub.
+
+### GitHub Actions Documentation
+Full documentation for GitHub Actions can be found on GitHub at [https://help.github.com/en/categories/automating-your-workflow-with-github-actions](https://help.github.com/en/categories/automating-your-workflow-with-github-actions)
+
+## Syntax
+Workflow files use YAML syntax, and must have either a .yml or .yaml file extension. You must store workflow files in the `.github/workflows` directory of your repository.
+
+### Workflow `name`
+The name of your workflow. GitHub displays the names of your workflows on your repository's actions page.
+
+### `on` Event
+Required The name of the GitHub event that triggers the workflow. You can provide a single event string, array of events, or an event configuration map that schedules a workflow or restricts the execution of a workflow to specific files, tags, or branch changes.
+
+```
+# Trigger on push
+on: push
+```
+
+```
+# Trigger the workflow on push or pull request
+on: [push, pull_request]
+```
+
+{% endcapture %}
+<div class="col-md-6">
+{{ colOne | markdownify }}
+</div>
+
+
+{% capture colTwo %}
+
+### `jobs`
+A workflow run is made up of one or more jobs. Jobs run in parallel by default. To run jobs sequentially, you can define dependencies on other jobs using the `jobs.<job_id>.needs` keyword. Each job runs in a fresh instance of the virtual environment specified by runs-on.
+
+### `runs-on`
```

Comment:
One thing we may want to do with commands after our jobs heading is make them level 4 headers #### because it continues the hierarchy that is set up in the yaml. runs-on for instance is actually jobs.<job_id>.runs-on in the documentation.

Thoughts?

Reply:
it ends up looking nicer with `<h4>` headings. marking this as resolved.